examples/22-windows/windows.cpp

```diff
@@ -167,17 +167,16 @@ class ExampleWindows : public entry::AppI
 
 	bool update() override
 	{
-		entry::WindowState state;
```

Comment:
Since mouse state is part of this, it was never getting carried over between frames. Resulted in flickering hovering in ImGui panels.